modules/govuk/templates/node/s_backend_lb/rate-limiting.conf.erb

```diff
@@ -26,3 +26,15 @@ limit_req_zone $post_limit_req zone=contact:5m rate=6r/m;
 # Return 429 (Too Many Requests) instead of the default 503.
 limit_req_status 429;
 limit_conn_status 429;
+
+# Create a rate limiting zone that handles limiting excessive load from
+# a specific high-load google apps script by ID. This may need to be changed to limit 
+# all Google-App-Script calls.
+map $http_user_agent $script_limit {
+  default "";
+  ~*(Google-Apps-Script) $binary_remote_addr;
```

Comment:
I think with `$binary_remote_addr` here we're just limiting by IP address, so this is adding nothing over existing configuration, I think we want `$http_user_agent `